packages/lesswrong/server/repos/BookmarksRepo.ts

```diff
@@ -0,0 +1,70 @@
+import { recordPerfMetrics } from "./perfMetricWrapper";
+import AbstractRepo from "./AbstractRepo";
+import { Bookmarks } from "../collections/bookmarks/collection";
+
+interface UltraFeedBookmark {
+  documentId: string;
+  collectionName: string;
+  postId?: string;
+  directChildrenCount?: number;
```

Comment:
I think this aren't optional but are `| null` (as with everything coming back directly from postgres).